src/Microsoft.AspNetCore.Localization/AcceptLanguageHeaderRequestCultureProvider.cs

```diff
@@ -33,7 +34,7 @@ public override Task<ProviderCultureResult> DetermineProviderCultureResult(HttpC
 
             if (acceptLanguageHeader == null || acceptLanguageHeader.Count == 0)
             {
-                return Task.FromResult((ProviderCultureResult)null);
+                return TaskCache<ProviderCultureResult>.DefaultCompletedTask;
```

Comment:
I'm not sure if this change applicable in this PR, or may we need another PR, while this one specially for adding `RouteDataCulureRequestProvider`